well-formed-strings-test.rb

```diff
@@ -0,0 +1,25 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'well-formed-strings.rb'
+
+class StringValidatorTest < Minitest::Test
+  def test_parens
+    v = StringValidator.new
+    assert v.validate("()"), true
+  end
+
+  def test_mess
+    v = StringValidator.new
+    assert v.validate("([{}[]])"), true
+  end
+
+  def test_one_off
+    v = StringValidator.new
+    assert v.validate("()["), false
+  end
+
+  def test_offset
```

Comment:
I appreciate the fact that you included sad path testing here. 

You were the first to do so. 

Brava!